dns/nextdns-community/src/opnsense/mvc/app/models/OPNsense/Nextdns/General.xml

```diff
@@ -0,0 +1,71 @@
+<model>
+    <mount>//OPNsense/nextdns/settingsCli</mount>
+    <description>Nextdns CLI configuration</description>
+    <version>0.0.1</version>
+    <items>
+        <enabled type="BooleanField">
+            <default>0</default>
+            <Required>Y</Required>
+        </enabled>
+
+        <email type="TextField">
```

Comment:
There's also EmailField available :)

Reply:
yes, this one needs to change to email, the password one needs to change to write-only text.